apps/frontend/src/components/ui/UserProfileBanner.tsx

```diff
@@ -27,57 +27,72 @@ export default function UserProfileBanner(props: {
     });
   };
   if (!props.user) return null;
-
   return (
     <Card className='flex max-md:flex-col md:flex-row max-md:items-center'>
-      <div className='min-w-44 min-h-44 w-1/4 h-full aspect-square relative'>
-        <Image
-          src='https://mozsarmate.me/marci.jpg'
-          placeholder='blur'
-          blurDataURL='data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mNkAAIAAAoAAv/lxKUAAAAASUVORK5CYII='
+      <div className='min-w-44 min-h-44 w-1/4 h-full relative'>
+        <img
+          src={`${process.env.NEXT_PUBLIC_API_URL}/users/${props.user.authSchId}/profile-picture`}
           alt='PROFIL KEP'
-          fill
           className='md:rounded-l-xl max-md:rounded-xl max-md:my-4'
         />
         <div className='w-full absolute flex bottom-2'>
-          <Button variant='secondary' className='block m-auto'>
+          <Link href='/profile/image' className='m-auto bg-white rounded p-2'>
             Profilkép módosítása
-          </Button>
+          </Link>
+          {/*<Dialog>
+            <DialogTrigger variant='secondary' className='block m-auto' asChild>
+              Profilkép módosítása
+            </DialogTrigger>
+            <DialogContent className='sm:max-w-[425px]'>
+              <DialogHeader>
+                <DialogTitle>Profil kép feltöltése</DialogTitle>
+                <DialogDescription>Válassz egy kiváló képet magadról!</DialogDescription>
+              </DialogHeader>
+              <DialogContent>
+                <input type='file' />
+              </DialogContent>
+              <DialogFooter>
+                <Button type='submit'>Tovább</Button>
+              </DialogFooter>
+            </DialogContent>
+          </Dialog>*/}
         </div>
       </div>
       <div className='w-full'>
-        <CardContent>
-          <div className='flex md:flex-row max-md:flex-col max-md:items-center max-md:gap-4 mt-10 justify-between'>
-            <div className='flex items-start'>
-              <div>
-                <TTitle className='mt-0'>{props.user!.fullName}</TTitle>
-                <Th2 className='ml-8'>{props.user!.neptun}</Th2>
+        <div className='w-full'>
+          <CardContent>
+            <div className='flex md:flex-row max-md:flex-col max-md:items-center max-md:gap-4 mt-10 justify-between'>
+              <div className='flex items-start'>
+                <div>
+                  <TTitle className='mt-0'>{props.user!.fullName}</TTitle>
+                  <Th2 className='ml-8'>{props.user!.neptun}</Th2>
+                </div>
+                <Badge className='text-sm px-4 py-2 rounded-xl' variant='secondary'>
+                  {props.user!.role}
+                </Badge>
               </div>
```

Comment:
we need some spacing between these two, and I think the roles should be translated

Reply:
I think the english version can stay, but the spacing is valid.